docs/extend/plugins_volume.md

```diff
@@ -140,15 +162,15 @@ at the first mount request and deprovision at the last corresponding unmount req
 **Response**:
 ```json
 {
-    "Mountpoint": "/path/to/directory/on/host",
+    "Mountpoint": "/path/to/directory/inside/plugin",
```

Comment:
What was originally there is v1. i.e "Mountpoint": "/path/to/directory/on/host" in v1

What's new is v2, i.e "Mountpoint": "/path/under/PropagateMount"